### DISCUSSION TITLE
Quad/OctoMiner communications?

### DISCUSSION THREAD

**Discussion by sburggsx** [BitMaker-hub/NerdMiner_v2#153] [General]
I see boards for sale on Tindie for quad and octo setups for 4 and 8 Nerdminers. Am I correct in assuming two Nerdminers cannot talk to each other? So the only benefit these boards have is mounting? There's no collaboration across boards?

**Comment by BitMaker-hub** [comment]
Hi, this board only offers better benefit for mounting. I'm preparing a small communication protocol between CPUs to be able to daisy chain them.

**Comment by mmilleror** [comment]
I just received my octo board. I hope to start populating it soon.

**Comment by dwightmulcahy** [comment]
I've been running an OctoBoard for a bit now.  The weak point is the USB-C connector, which I broke off within a day of using it.   I suggest using epoxy glue on the USB-C connector to strengthen that weak point. 

:(

It would be cool if there were an advantage to these besides just mounting/sharing power.
